src/commands/game_options/timer.ts

```diff
@@ -199,6 +204,39 @@ export default class GuessTimeoutCommand implements BaseCommand {
                 )} | Guess timeout disabled.`,
             );
         } else {
+            timer = Math.round(timer! * 100) / 100;
+            if (timer < GuessTimeoutCommand.TIMER_MIN_VALUE_NON_CLIP) {
+                if (session) {
+                    if (!session.isGameSession() || !session.isClipMode()) {
```

Comment:
Doesn't `!session.isGameSession()` imply `!session.isClipMode()`?

Reply:
This code branch is entered when not a game session, or a non-clip game mode is being used